cmd/wol/wol.go

```diff
@@ -5,15 +5,14 @@ package main
 import (
 	"errors"
 	"fmt"
+	"github.com/sabhiram/go-wol/pkg/wol"
```

Comment:
nit : I always try to keep non standard packages after the first set of imports - but we can deal w/ this later 👍 Thank you for this work btw!

Reply:
Sorry about this, I usually run gofmt over my files and it rearranges imports like this.

I can revert that if you want!